criu/config.c

```diff
@@ -648,13 +650,17 @@ int parse_options(int argc, char **argv, bool *usage_error,
 					goto bad_arg;
 
 				*aux = '\0';
-				if (veth_pair_add(optarg, aux + 1))
+				if (veth_pair_add(optarg, aux + 1)) {
+					pr_err("veth_pair_add failed.\n");
 					return 1;
```

Comment:
`veth_pair_add()` will report an allocation error or the error occurred during further processing. I didn't check all the paths, but those I've checked have error reporting.